duckduckgo/iOS
# Change App Icon from Settings

Task/Issue URL: https://app.asana.com/0/1148564399326804/1154069155236875

**Description**:
Added option to change app icons from Settings.

**Steps to test this PR**:
1. Open app on iOS 10.3 or higher
2. Go to Settings->App Icon
3. Select icon
4. Icon of the app should be changed

Optional
1. Open app on iOS 10.2 or lower
2. Go to Settings
3. No "App Icon" setting should be present
---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
